jlh/WeatherApp/app/src/main/java/com/e_conomic/weatherapp/domain/mappers/ForecastDataMapper.kt

```diff
@@ -9,19 +9,34 @@ import com.e_conomic.weatherapp.domain.model.Forecast as ModelForecast
 
 class ForecastDataMapper {
 
-    fun convertFromDataModel(forecast: ForecastResult): ForecastList {
+    companion object {
+        val NO_ICON = "ForecastDataMapper:noIcon"
+        val NO_DESCRIPTION = "ForecastDataMapper:noDescription"
```

Comment:
This is the convention for such string values he uses in the book - do you have a convention for this? @tudorgk @AndersHqst